ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,532 @@
+- [eos_config_deploy_eapi](https://avd.sh/en/latest/roles/eos_config_deploy_eapi/index.html) which deploys the EOS configuration directly to Arista devices using config replace. This option provides no change control and no rollback capability.
+
+## Source of Truth
+
+In a legacy network where configuration is not administered centrally,
+you have very little control of the relationship between the configuration you *intend* to be applied to the network,
+and the configuration actually *running* on the network. You might have centralized low level design documents describing in great detail
+how exactly the network should function, but you don't have much but best intentions to ensure that your **entire** network is working
+as you intended.
+
+It takes only a single configuration mistake on a single device to create havoc.
+
+Since operating many networking devices also typically result in having many networking engineers, 
+there is even more room for error. Different people do things differently and repetitive
+tasks are not always executed in the exact same manner.
+
+With Ansible AVD, you define not only the topology of your network centrally, but also which services are used where in a central repository
+of text files.
+
+The “source of truth” exists inside Ansible AVD at any point in time, meaning you have a full overview of your entire designed
+network configuration without having to look at individual network devices.
+An additional benefit of Ansible AVD is that by design you have an always available backup of your network configuration
+including automated documentation in markdown format.
+No longer do you have to remember to update documentation about which interface is connected to which device etc.,
+whenever you change something on a device.
+It's all done automatically based on the configuration built and applied to the network devices by Ansible, everytime you execute the playbooks.
+
+Below you will find two examples of documentation automatically created by Ansible AVD:
+
+- [5 stage CLOS fabric across two Data Centers](../molecule/eos_designs-twodc-5stage-clos/documentation/fabric/TWODC_5STAGE_CLOS-documentation.md)
+- [A single leaf inside the fabric above](../molecule/eos_designs-twodc-5stage-clos/documentation/devices/DC1-POD1-L2LEAF1A.md)
+
+## How do I use Ansible AVD?
+
+Ansible AVD comes with pre-built templates that you can either decide to re-use as-is or adapt to your liking.
+
+Once the templates reflect your desired network configuration, you deploy the configuration either directly to the Arista EOS devices or via Arista Cloudvision using its internal change management system, where the changes can be reviewed, approved and executed in accordance with existing company requirements.
+The deployment is executed typically using CLI commands or via the GUI available with Ansible Tower. 
+
+For example, running the CLI command: 
+```
```

Comment:
Blank line before the quotes, this will throw [md031](https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md031) error

Reply:
Another error, [md040](https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md040)

add ```shell``` after the triple quotes like ```shell